api/src/main/java/fade/mirror/Invokable.java

```diff
@@ -40,4 +56,6 @@
      */
     @Contract(pure = true)
     @NotNull Class<T> getReturnType();
+
+    Class<?>[] getParameterTypes();
```

Comment:
The `Parameterized` interface already offers a `getParameterTypes()` method (which returns `MClass`es). Should we merge the `Parameterized` and `Invokable` interfaces, or is there a case where something can have parameters but is not executeable? This might need to be checked.